etcdmain/etcd.conf.sample.yml

```diff
@@ -0,0 +1,135 @@
+# This is the configuration file for the etcd server.
+
+# Human-readable name for this member.
+name: 
```

Comment:
I had mostly set everything to the flag default values. Had missed out this one. Will set it.